librocksdb-sys/build/vendor.rs

```diff
@@ -0,0 +1,331 @@
+use std::env;
+
+#[cfg(any(
+    feature = "bzip2",
+    feature = "lz4",
+    feature = "snappy",
+    feature = "zlib",
+    feature = "zstd"
+))]
+fn enforce_rerun<P: AsRef<std::path::Path>>(path: P) {
+    println!("cargo:rerun-if-changed={}", path.as_ref().to_string_lossy());
+}
+
+#[cfg(any(
+    feature = "bzip2",
+    feature = "lz4",
+    feature = "snappy",
+    feature = "zlib",
+    feature = "zstd"
+))]
+fn check_submodule<P: AsRef<std::path::Path>>(path: P) {
+    let path = path
+        .as_ref()
+        .canonicalize()
+        .unwrap_or_else(|_| panic!("Failed to canonicalize {:?}", path.as_ref()));
+    let dir =
+        std::fs::read_dir(&path).unwrap_or_else(|_| panic!("Failed to open directory {:?}", path));
+    if dir.count() == 0 {
+        println!(
```

Comment:
Perhaps `eprintln` would be more idiomatic?

Reply:
Yeah :)